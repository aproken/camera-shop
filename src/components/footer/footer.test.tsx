import { render, RenderResult } from '@testing-library/react';
import Footer from './footer';
import { BrowserRouter as Router } from 'react-router-dom';


describe('Component: Footer', () => {
  it('Тестирование компонента Footer', () => {
    const { container }: RenderResult = render(
      <Router>
        <Footer />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
