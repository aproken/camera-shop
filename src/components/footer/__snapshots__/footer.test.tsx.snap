// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Footer Тестирование компонента Footer 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <div
        class="footer__info"
      >
        <a
          aria-label="Переход на главную"
          class="footer__logo"
          href="/"
        >
          <svg
            aria-hidden="true"
            height="36"
            width="100"
          >
            <use
              xlink:href="#icon-logo-mono"
            />
          </svg>
        </a>
        <p
          class="footer__description"
        >
          Интернет-магазин фото- и видеотехники
        </p>
        <ul
          class="social"
        >
          <li
            class="social__item"
          >
            <a
              aria-label="Переход на страницу "
              class="link"
              href="/#todo"
            >
              <svg
                aria-hidden="true"
                height="20"
                width="20"
              >
                <use
                  xlink:href="#icon-vk"
                />
              </svg>
            </a>
          </li>
          <li
            class="social__item"
          >
            <a
              aria-label="Переход на страницу "
              class="link"
              href="/#todo"
            >
              <svg
                aria-hidden="true"
                height="20"
                width="20"
              >
                <use
                  xlink:href="#icon-vk"
                />
              </svg>
            </a>
          </li>
          <li
            class="social__item"
          >
            <a
              aria-label="Переход на страницу "
              class="link"
              href="/#todo"
            >
              <svg
                aria-hidden="true"
                height="20"
                width="20"
              >
                <use
                  xlink:href="#icon-vk"
                />
              </svg>
            </a>
          </li>
        </ul>
      </div>
      <ul
        class="footer__nav"
      >
        <li
          class="footer__nav-item"
        >
          <p
            class="footer__title"
          >
            Навигация
          </p>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                Каталог
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                Гарантии
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                Доставка
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                О компании
              </a>
            </li>
          </ul>
        </li>
        <li
          class="footer__nav-item"
        >
          <p
            class="footer__title"
          >
            Ресурсы
          </p>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                Курсы операторов
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                Блог
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                Сообщество
              </a>
            </li>
          </ul>
        </li>
        <li
          class="footer__nav-item"
        >
          <p
            class="footer__title"
          >
            Поддержка
          </p>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                FAQ
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/#todo"
              >
                Задать вопрос
              </a>
            </li>
          </ul>
        </li>
      </ul>
    </div>
  </footer>
</div>
`;
